index.html

```diff
@@ -0,0 +1,306 @@
+
+                <br>
+
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Oracle Netsuite, SuiteScript</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Bitrix</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Magento</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> WordPress</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Elementor</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Hospedagem</p>
+                </div>
+                
+            </div>
+
+            <hr>
+
+            <div id="skills-tools-block">
+
+                <h2>BI (Business Intelligence):</h2>
+
+                <br>
+
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Looker Studio</p>
+                </div>
+                
+            </div>
+
+            <hr>
+            
+            <!-- Botão para baixar currículo -->
+            <button class="download-cv-btn">
+                <a href="./curriculo/Currículo CV Básico Minimalista Simples Preto e Branco (1).pdf" download="Thálisson Gonçalves - Currículo">
+                    DOWNLOAD CURRÍCULO 
+                    <i class="fa-solid fa-download" style="color: #8c8c8e"></i>
+                </a>
+            </button>
+            
+        </div>
+
+        <div id="social-block">
+            <a href="https://www.linkedin.com/in/thalissongoncalves/" target="_blank"><i class="fa-brands fa-linkedin"></i></a>
+            <a href="https://github.com/thalissongoncalves" target="_blank"><i class="fa-brands fa-github"></i></a>
+            <a href="https://www.instagram.com/thalissonzpyg/" target="_blank"><i class="fa-brands fa-instagram"></i></a>
+        </div>
+        
+    </div>
+
+    
+
+    <div id="menu-block">
+        <div class="menu-btn-block">
+            <button class="menu-btn"><i class="fa-solid fa-bars fa-2xl" style="color: #8c8c8e;"></i></button>
+        </div>
+    </div>
+
+    <div id="main-block">
+
+        <div class="main-message">
+            <h2>Olá! Muito prazer! </h2>
+            <h1>Seja bem-vindo ao meu portfólio!</h1>
+            <h3 class="typing-animation">Desenvolvedor Full-Stack - MG / BRASIL</h3>
+            <div class="button-mode">
+                <button class="theme-btn"><i class="fa-solid fa-sun" style="color: black"></i><br><p>White mode</p></button>
+                <button class="lenguage-btn"><i class="fa-solid fa-flag-usa" style="color: black"></i><br><p>English mode</p></button>
+            </div>
```

Comment:
`<br>` dentro de um button nao e legal, normalmente usamos apenas textos ou icones dentro. Aplique o estilo que precisa usando css, voce pode simular a quebra de linha usando flexbox ou grid.

```suggestion
            <div class="button-mode">
                <button class="theme-btn"><i class="fa-solid fa-sun" style="color: black"></i><p>White mode</p></button>
                <button class="lenguage-btn"><i class="fa-solid fa-flag-usa" style="color: black"></i><p>English mode</p></button>
            </div>
```